src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -40,6 +41,16 @@
 	 */
 	public static final String KUBERNETES_DEPLOYMENT_NODE_SELECTOR = "spring.cloud.deployer.kubernetes.deployment.nodeSelector";
 
+	private Config fabric8 = new Config();
```

Comment:
`new Config()` is deprecated

Reply:
@chrisjs  Yes, I saw that when I had it in. The deprecation message says this: 
```
//In future releases (2.4.x) the public constructor will be empty.
//The current functionality will be provided by autoConfigure().
//This is a necessary change to allow us distinguish between auto configured values and builder values.
```
We'll use autoConfigure() once we switch to the updated client. Sounds ok?

Reply:
per slack discussion, lets avoid calling deprecated methods